### DISCUSSION TITLE
What does the "LENS VIEW" do?

### DISCUSSION THREAD

**Discussion by franva** [upscayl/upscayl#723] [Q&A] [answered]
I just found the new features in version 2.9.9 and good job :)

I found on the new side panel, there are options, e.g. 
1. Lens View
2. Slider View
3. Zoom Amount
4. Lens Size

![image](https://github.com/upscayl/upscayl/assets/5151146/3de7faf0-ef92-43a7-8ee0-74b541f1b5f4)

I have loaded an image and fiddled around these controls, but cannot see any changes accordingly.

Also searched the entire repositories across the organization and only found 5 places in code. But no documents explain what do they do and how do they work.
![image](https://github.com/upscayl/upscayl/assets/5151146/26598514-3f2b-43c2-a6b3-0433a6ebe024)

Appreciate if someone could explain these options in any forms.

**Comment by aaronliu0130** [comment, marked as answer]
Try flipping that toggle to slider view and back again. You need to Upscayl an image first to get a comparison.

**Reply by franva** [reply]
thanks, I can see what they do.
